Source: written by Pierre Lejeune <superheron@gmail.com>, for Exherbo
Upstream: no
Reason: Respect non-symlinked libdir

--- a/files/usr/share/cinnamon/cinnamon-settings/bin/capi.py	2022-11-04 16:29:06.957364077 +0100
+++ b/files/usr/share/cinnamon/cinnamon-settings/bin/capi.py	2022-09-05 16:15:23.000000000 +0200
@@ -31,7 +31,8 @@
         # get the arch-specific triplet, e.g. 'x86_64-linux-gnu' or 'arm-linux-gnueabihf'
         # see also: https://wiki.debian.org/Python/MultiArch
         triplet = sysconfig.get_config_var('MULTIARCH')
-        paths = ["/usr/lib", f"/usr/lib/{triplet}"]
+        libdir = sysconfig.get_config_var('LIBDIR')
+        paths = [libdir, "/usr/lib", f"/usr/lib/{triplet}"]
 
         # On x86 archs, iterate through multiple paths
         # For instance, on a Mint i686 box, the path is actually /usr/lib/i386-linux-gnu
